### PR TITLE
feat: toast component implemented

### PR DESCRIPTION
**Pull request by govardhanreddyc79**

### CODE REVIEW

**Review by aurobindoGupta — APPROVED** [review]

File: Components/Toast/toast.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
+
+.element-display {
+    display: flex;
+    justify-content: center;
+    flex-direction: row;
```
Comment (rohanmathur91) [inline Components/Toast/toast.css:5]: default direction is row so no need to set

File: Components/Toast/toast.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    flex-wrap: wrap;
+    align-items: center;
+    padding: 1rem;
+    border: 1px solid;
+    border-radius: 10px;
+    gap:15px;
```
Comment (rohanmathur91) [inline Components/Toast/toast.css:11]: use rem if possible

**Review by rohanmathur91 — APPROVED** [review]
Looks good to me.

**Review by saicharan-gouru — APPROVED** [review]
Looks good to me.

File: Components/Toast/toast.css (hunk 2 above)
Comment (govardhanreddyc79) [reply Components/Toast/toast.css:11]: sure.

File: Components/Toast/toast.css (hunk 1 above)
Comment (govardhanreddyc79) [reply Components/Toast/toast.css:5]: okay, Thanks

**Pull request merged**